x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/expression/NameId.java

```diff
@@ -47,4 +47,8 @@ public boolean equals(Object obj) {
     public String toString() {
         return Long.toString(id);
     }
+
+    public long toLong() {
```

Comment:
I'd like to not open up this class and rely on it. Especially since the motivation is to get the id which is then converted to a string in a name - just call NameId#toString().